External test for OpenZeppelin failing on legacy codegen at `ERC20VotesTimestampMock`

`t_native_test_ext_zeppelin` started failing recently in optimized legacy runs.

### Details of the failure
[run 1265534 of `t_native_test_ext_zeppelin`](https://app.circleci.com/pipelines/github/ethereum/solidity/28507/workflows/cef6dc8d-1e7a-48db-be1e-82cb157e399d/jobs/1265534):

```
1) Contract: Governor
       using $ERC20VotesTimestampMock
         state
           Succeeded:

      AssertionError: expected '4' to equal '1'
      + expected - actual

      -4
      +1
      
      at Context.<anonymous> (test/governance/Governor.test.js:382:75)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

--- a/test/externalTests/zeppelin.sh
+++ b/test/externalTests/zeppelin.sh
@@ -138,6 +138,10 @@ function zeppelin_test
     sed -i "s|describe(\('reverting initialization'\)|describe.skip(\1|g" test/proxy/Proxy.behaviour.js
     sed -i "s|it(\('does not allow remote callback'\)|it.skip(\1|g" test/security/ReentrancyGuard.test.js
 
+    # TODO: Remove this when https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4049 gets fixed
+    sed -i "s|it(\('Succeeded'\)|it.skip(\1|g" test/governance/Governor.test.js
+    sed -i "s|it(\('Delay is extended to prevent last minute take-over'\)|it.skip(\1|g" test/governance/extensions/GovernorPreventLateQuorum.test.js
+
     neutralize_package_json_hooks
     force_hardhat_compiler_binary "$config_file" "$BINARY_TYPE" "$BINARY_PATH"
     force_hardhat_compiler_settings "$config_file" "$(first_word "$SELECTED_PRESETS")"

